core/src/main/resources/error/error-classes.json

```diff
@@ -157,6 +157,11 @@
     ],
     "sqlState" : "22018"
   },
+  "CANNOT_PARSE_INTERVAL" : {
+    "message" : [
+      "Unable to parse '<delayThreshold>'. Please ensure that the value provided is in a valid format for defining an interval. You can reference the documentation for the correct format. If the issue persists, please double check that the input value is not null or empty and try again."
```

Comment:
This is a top-level error class and we should be a bit more general. `delayThreshold` -> `intervalString`?